jenkins-jobs/gen3-run-load-tests.groovy

```diff
@@ -68,13 +68,21 @@ pipeline {
         }
         stage('Setup for Load Tests') {
             steps {
+                withCredentials(file(credentialsId: 'qa-dcp-credentials-json', variable: 'QA_DCP_CREDS_JSON')){
                 sh """#!/bin/bash
 
                   export KUBECTL_NAMESPACE="$TARGET_ENVIRONMENT"
                   # setup gen3 CLI
                   export GEN3_HOME=\$WORKSPACE/cloud-automation
                   source \$GEN3_HOME/gen3/gen3setup.sh
 
+                  indexdRecord=$(curl -s GET 'https://${KUBECTL_NAMESPACE}/index/index' | jq -r '.records | length)
```

Comment:
moving the code inside the `[ "$LOAD_TEST_DESCRIPTOR" == "fence-presigned-url" ]` block , so that indexd records are created only when the presigned url load tests are needed to executed.